precompiles/ArbOwner.go

```diff
@@ -287,6 +287,17 @@ func (con ArbOwner) SetWasmBlockCacheSize(c ctx, _ mech, count uint16) error {
 	return params.Save()
 }
 
+// SetMaxWasmSize sets the maximum size the wasm code can be in bytes after
+// decompression.
+func (con ArbOwner) SetMaxWasmSize(c ctx, _ mech, maxWasmSize uint32) error {
```

Comment:
should only exist for arbos >= 40

Reply:
I added the exposing of it to precompiles.go. Is that sufficient, or are you suggesting that I should also guard the body of this method with an arbos.Version >= 40 check?

Reply:
Looks like it was sufficient to only expose it in arbos version >= 40  in precompiles.go.